src/test/java/com/google/ehub/servlets/ItemPageServletTest.java

```diff
@@ -6,15 +6,18 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-import com.google.common.collect.Lists;
 import com.google.appengine.api.datastore.DatastoreService;
```

Comment:
Ideally, we should also add some tests here for retrieving comments that don't belong to the current user and for retrieving comments while the user is logged out. But we can punt it to the following PR that includes ItemPageServlet code changes for deleting items.

Reply:
Noted